docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -196,3 +199,12 @@ and a file system location of ``/var/my_tables/test_table``::
         format = 'PARQUET',
         partitioning = ARRAY['c1', 'c2'],
         location = '/var/my_tables/test_table')
+
+Limitations
```

Comment:
Remove this whole Limitations sections .. I will submit a PR shortly that adds a `SQL support` section and it we will link to all supported statements and add that no other statements are supported. I will work with this next week with @Jessie212

Reply:
This is now implemented and the limitation no longer applies.